### ISSUE TITLE
Separated releases

### ISSUE THREAD

**Issue by sagitter** [copasi/copasi-dependencies#8]
Hello!

Please, can you create a release when new versions of (for example) libCombine or Zipper are ready?

**Comment by fbergmann** [comment]
Of course! The release here was just so that we have some files to speed up CI builds.
